config/templates/cluster.yaml

```diff
@@ -169,8 +169,10 @@ controller:
 # subnets:
 #   - availabilityZone: us-west-1a
 #     instanceCIDR: "10.0.0.0/24"
+#     subnetId: "subnet-xxxxxxxx" #optional
```

Comment:
ideally it should work with just `subnetId` specified
```
subnets:
 - subnetId: subnet-aaaaa
 - subnetId: subnet-zzzzz
```

does it work like that already?